### PR TITLE
Required changes after adding annotations

### PR DESCRIPTION
**Pull request by artek-koltun**
See https://github.com/opiproject/opi-api/pull/321

@glimchb @sburla-marvell 
Please pay attention to 799d7b34e5acecda48c25c48bf775b9c4024725f. Apparently, by mistake, `cntlid` was used as a part of SPDK create controller response, but a correct field should be `ctrlr_id` Can you confirm?

Also, looks like xPU supports both: auto controller id assignment and requesting a specific id. I checked pdf in the repo, but please confirm that -1 would auto assign id

### CODE REVIEW

File: pkg/frontend/nvme_controller.go [hunk 1]
```diff
@@ -76,12 +79,16 @@ func (s *Server) CreateNvmeController(_ context.Context, in *pb.CreateNvmeContro
 		return nil, err
 	}
 
+	ctrlrID := autoCtrlrIDAllocation
+	if in.NvmeController.Spec.NvmeControllerId != nil {
+		ctrlrID = int(*in.NvmeController.Spec.NvmeControllerId)
+	}
```
Comment (glimchb) [inline pkg/frontend/nvme_controller.go:85]: maybe this needs to be in the https://github.com/opiproject/opi-marvell-bridge/commit/799d7b34e5acecda48c25c48bf775b9c4024725f commit ?

Comment (artek-koltun) [reply pkg/frontend/nvme_controller.go:85]: In this case, it will be better to put in a separate commit then: f04093df21190eba68623e62280e61033ff6344f

**Review by glimchb — APPROVED** [review]

**Review by sburla-marvell — APPROVED** [review]

**Comment by artek-koltun** [comment]
> See [opiproject/opi-api#321](https://github.com/opiproject/opi-api/pull/321)
> 
> @glimchb @sburla-marvell Please pay attention to [799d7b3](https://github.com/opiproject/opi-marvell-bridge/commit/799d7b34e5acecda48c25c48bf775b9c4024725f). Apparently, by mistake, `cntlid` was used as a part of SPDK create controller response, but a correct field should be `ctrlr_id` Can you confirm?
Yes. This is correct. It should be ctrlr_id
> 
> Also, looks like xPU supports both: auto controller id assignment and requesting a specific id. I checked pdf in the repo, but please confirm that -1 would auto assign id
Yes Marvell DPU supports both auto assignment and requesting a specific id. Passing -1 will auto assign the id

Waiting for runner release to get ci passed https://github.com/actions/runner-images/issues/8080

**Comment by glimchb** [comment]
@sburla-marvell thanks for confirming
@artek-koltun thanks for fixing and merging!

**Pull request merged**